demo/index.html

```diff
@@ -17,101 +17,73 @@
-          <paper-toggle-button checked>Hydrogen</paper-toggle-button>
-          <paper-toggle-button checked>Nitrogen</paper-toggle-button>
-          <paper-toggle-button checked>Calcium</paper-toggle-button>
-        </div>
-      </div>
+    <div class="vertical-section-container centered">
+      <h3>Toggle buttons can be checked and disabled</h3>
+      <demo-snippet class="centered-demo">
+        <template>
+          <paper-toggle-button>Toggle</paper-toggle-button>
+          <paper-toggle-button checked>Toggle</paper-toggle-button>
+          <paper-toggle-button disabled>Disabled</paper-toggle-button>
+        </template>
+      </demo-snippet>
+    </div>
 
-      <div>
-        <h4>Disabled</h4>
-        <div class="horizontal-section">
-          <paper-toggle-button disabled>Oxygen</paper-toggle-button>
-          <paper-toggle-button disabled>Carbon</paper-toggle-button>
-          <paper-toggle-button checked disabled>Hydrogen</paper-toggle-button>
-          <paper-toggle-button checked disabled>Nitrogen</paper-toggle-button>
-          <paper-toggle-button checked disabled>Calcium</paper-toggle-button>
-        </div>
-      </div>
+    <div class="vertical-section-container centered">
+      <h3>Toggle buttons can hide the ripple effect using the <i>noink</i> attribute</h3>
+      <demo-snippet class="centered-demo">
+        <template>
+          <paper-toggle-button noink>Toggle</paper-toggle-button>
+        </template>
+      </demo-snippet>
+    </div>
 
-      <div>
-        <h4>Color</h4>
-        <div class="horizontal-section">
-          <paper-toggle-button class="blue">Oxygen</paper-toggle-button>
-          <paper-toggle-button class="red">Carbon</paper-toggle-button>
-          <paper-toggle-button checked class="orange">Hydrogen</paper-toggle-button>
-          <paper-toggle-button checked class="green">Nitrogen</paper-toggle-button>
-          <paper-toggle-button checked class="blue">Calcium</paper-toggle-button>
-        </div>
-      </div>
+    <div class="vertical-section-container centered">
+      <h3>Toggle buttons can be styled using custom properties</h3>
+      <demo-snippet class="centered-demo">
+        <template>
+          <paper-toggle-button class="green">Toggle</paper-toggle-button>
+          <paper-toggle-button checked class="pink">Toggle</paper-toggle-button>
 
-      <div>
-        <h4>Noink</h4>
-        <div class="horizontal-section">
-          <paper-toggle-button noink>Oxygen</paper-toggle-button>
-          <paper-toggle-button noink>Carbon</paper-toggle-button>
-          <paper-toggle-button checked noink>Hydrogen</paper-toggle-button>
-          <paper-toggle-button checked noink>Nitrogen</paper-toggle-button>
-          <paper-toggle-button checked noink>Calcium</paper-toggle-button>
-        </div>
-      </div>
+          <style is="custom-style">
+            paper-toggle-button.green {
+              --paper-toggle-button-checked-bar-color:  var(--paper-green-500);
```

Comment:
how about making this super colourful, and instead of dark green using teal (so teal to green), and instead of the dark pink, the indigo (so indigo to pink)? :)